### DISCUSSION TITLE
Does ClearURL replace "Don't track me Google"?

### DISCUSSION THREAD

**Discussion by Oymate** [ClearURLs/Addon#147] [Q&A] [answered]
Link:https://addons.mozilla.org/en-US/firefox/addon/dont-track-me-google1/

**Comment by Splinter7149** [comment]
welp.... I would also like to know. I got my hopes up when I saw someone else had asked the same question when googling

**Comment by KevinRoebert** [comment, marked as answer]
Yes. ClearURLs does replace "Don't track me Google". See https://gitlab.com/ClearURLs/rules/-/blob/master/data.min.json#L163
